src/OrchardCore.Modules/OrchardCore.ContentTypes/Controllers/AdminController.cs

```diff
@@ -164,7 +164,14 @@ public async Task<ActionResult> Edit(string id)
         }
 
         typeViewModel.Editor = await _contentDefinitionDisplayManager.BuildTypeEditorAsync(typeViewModel.TypeDefinition, _updateModelAccessor.ModelUpdater);
-
+        if (String.Equals(submit, "save", StringComparison.Ordinal))
+        {
+            return RedirectToAction(nameof(List));
+        }
+        else if (String.Equals(submit, "saveAndContinue", StringComparison.Ordinal))
+        {
+            await _notifier.SuccessAsync(H["Admin node updated successfully."]);
+        }
```

Comment:
```suggestion
        if (string.Equals(submit, "saveAndContinue", StringComparison.Ordinal))
        {
            await _notifier.SuccessAsync(H["Admin node updated successfully."]);
            return View(typeViewModel);
        }
        else
        {
            return RedirectToAction(nameof(List));
        }
```

Reply:
This isn't done. Is this deliberate? And since actually `==` already uses Ordinal string comparison, I also updated it.